src/uwtools/api/config.py

```diff
@@ -45,10 +45,9 @@ def get_fieldtable_config(
     """
     Get a ``FieldTableConfig`` object.
 
-    :param config: FieldTable file to load (``None`` or unspecified => read ``stdin``), or initial
-        ``dict``
+    :param config: FieldTable file (``None`` => read ``stdin``), or initial ``dict``.
```

Comment:
In several API modules, I tried to apply consistent punctuation in docstrings, and made some parameter descriptions more concise.